solutions/common-shared-flows/logging-v1/sharedflowbundle/policies/Callout.Logging.xml

```diff
@@ -0,0 +1,36 @@
+<ServiceCallout name="Callout.Logging">
+  <Request>
+    <IgnoreUnresolvedVariables>true</IgnoreUnresolvedVariables>
+    <Set>
+      <Payload contentType="application/json">
+{
+  "organization": "{organization.name}",
+  "environment": "{environment.name}",
+  "apiProxy": "{apiproxy.name}",
+  "apiProxyRevision": "{apiproxy.revision}",
+  "apiProduct": "{apiproduct.name}",
+  "developerApp": "{apiproduct.name}",
+  "clientId": "{client_id}",
+  "developerId": "{developer.id}",
+  "requestUrl": "{request.url}",
+  "verb": "{request.verb}",
+  "correlationId": "{messageid}",
+  "proxyRequestReceived": "{client.received.end.timestamp}",
+  "proxyResponseSent": "{client.sent.end.timestamp}",
+  "targetResponseReceived": "{target.received.end.timestamp}",
+  "targetRequestSent": "{target.sent.end.timestamp}",
+  "targetResponseCode": "{message.status.code}",
+  "proxyResponseCode": "{response.status.code}",
+  "faultName":"{fault.name}",
+  "additionalParams": {
+  }
+}
+      </Payload>
+      <Verb>PUT</Verb>
+      <Path>/</Path>
+    </Set>
+  </Request>
+  <HTTPTargetConnection>
+    <URL>http://{custom.env.loggingUrl}</URL>
```

Comment:
Should this be simplified to include the scheme (http/https) in the KVM value instead of defining the URL partially here and partially in KVM?

Reply:
sadly not possible as you get a deployment error